doc/user_manual/generated/generateRomDoc.py

```diff
@@ -775,17 +776,20 @@
 internalRom = ''
 sklROM = ''
 dnnRom = ''
+# base classes first
 for name in orderedValidRom:
   if name in excludeObj:
     continue
   if name in validDNNRom:
     continue
   obj = SupervisedLearning.factory.returnClass(name)
   specs = obj.getInputSpecification()
+  romDescr = wrapText(specs.description, ' ')
```

Comment:
I checked the manual, it seems this information is already included. I think this line is not needed.